client/src/components/DishCard.js

```diff
@@ -20,28 +26,150 @@ const useStyles = makeStyles({
   info:{
     display: 'flex',
     flexDirection:"column",
-    padding:40,
+    paddingLeft:40,
+    paddingTop:40,
+    paddingBottom:40,
+  },
+
+  pplnum:{
+    backgroundColor: "#FF743D",
+    color:"white",
+    padding:"10px",
+    display:"inline",
+    justifyContent: "space-between",
+  },
+  textField:{
+    fullWidth:"true",
+  },
+  input:{
+    display:"none",
+  },
+  dishpic:{
+    cursor:"pointer",
   }
+
 });
 
 export default function SimpleCard() {
   const classes = useStyles();
 
+  const handleChange = name => event => {
```

Comment:
You can use one function that gets the name from an attribute on the target input element for this. First we'd need to add name attributes to each input (e.g. price input would get name="price"), then modify handleChange to look like:

handleChange = event => {setValues({...values, [event.target.name] : event.target.value});